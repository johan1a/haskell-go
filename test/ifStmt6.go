package main;
func main() {
if true {
    print( 1 )
} else {
    print ( 2 )
}
}
