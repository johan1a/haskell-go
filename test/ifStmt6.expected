[TopLevelDecl2 (FunctionDecl2 "main" Signature1 (Block [IfStmt (Ifstmt2 (BoolExpr True) (Block [SimpleStmt (ExpressionStmt (PrintCall [Num 1]))]) (Else2 (Block [SimpleStmt (ExpressionStmt (PrintCall [Num 2]))])))]))]
